admin/google_search_console/class-gsc.php

```diff
@@ -0,0 +1,175 @@
+		require_once 'views/gsc-display.php';
+	}
+
+	/**
+	 * Function that is triggered when the redirect page loads
+	 */
+	public function page_load() {
+		$this->service = new WPSEO_GSC_Service( WPSEO_GSC_Settings::get_profile() );
+
+		$this->request_handler();
+
+		add_action( 'admin_enqueue_scripts', array( $this, 'page_scripts' ) );
+	}
+
+	/**
+	 * Display the table
+	 *
+	 * @param WPSEO_GSC_Platform_Tabs $platform_tabs
+	 */
+	public function display_table( WPSEO_GSC_Platform_Tabs $platform_tabs ) {
+		// The list table.
+		$list_table = new WPSEO_GSC_Table( $platform_tabs, $this->service );
+		$list_table->prepare_items( );
+		$list_table->search_box( __( 'Search', 'wordpress-seo' ), 'wpseo-crawl-issues-search' );
+		$list_table->display();
+	}
+
+	/**
+	 * Load the admin redirects scripts
+	 */
+	public function page_scripts() {
+		wp_enqueue_script( 'wp-seo-admin-gsc', plugin_dir_url( WPSEO_FILE ) . 'js/wp-seo-admin-gsc' . WPSEO_CSSJS_SUFFIX . '.js', array( 'jquery' ), WPSEO_VERSION );
+		add_screen_option( 'per_page', array(
+			'label'   => __( 'Crawl errors per page', 'wordpress-seo' ),
+			'default' => 50,
+			'option'  => 'errors_per_page',
+		) );
+
+		wp_enqueue_style( 'jquery-qtip.js', plugins_url( 'css/jquery.qtip' . WPSEO_CSSJS_SUFFIX . '.css', WPSEO_FILE ), array(), WPSEO_VERSION );
+		wp_enqueue_style( 'metabox-tabs', plugins_url( 'css/metabox-tabs' . WPSEO_CSSJS_SUFFIX . '.css', WPSEO_FILE ), array(), WPSEO_VERSION );
+		wp_enqueue_script( 'jquery-qtip', plugins_url( 'js/jquery.qtip.min.js', WPSEO_FILE ), array( 'jquery' ), WPSEO_VERSION, true );
+	}
+
+	/**
+	 * Set the screen options
+	 *
+	 * @param string $status
+	 * @param string $option
+	 * @param string $value
+	 *
+	 * @return mixed
+	 */
+	public function set_screen_option( $status, $option, $value ) {
+		if ( 'errors_per_page' == $option ) {
+			return $value;
+		}
+	}
+
+	/**
+	 * Handles the POST and GET requests
+	 */
+	private function request_handler() {
+
+		// List the table search post to a get.
+		$this->list_table_search_post_to_get();
+
+		// Catch the authorization code POST.
+		$this->catch_authentication_post();
+
+		// Is there a reset post than we will remove the posts and data.
+		if ( filter_input( INPUT_POST, 'gsc_reset' ) ) {
+			$this->add_notification( __( 'The Google Webmaster Tools data has been removed. You will have to re-authenticate if you want to retrieve the data again.', 'wordpress-seo' ), 'update' );
+			WPSEO_GSC_Settings::clear_data( $this->service );
+		}
+
+		// Reloads al the issues.
+		if ( wp_verify_nonce( filter_input( INPUT_POST, 'reload-crawl-issues-nonce' ), 'reload-crawl-issues' ) && filter_input( INPUT_POST, 'reload-crawl-issues' ) ) {
+			// Reloading all the issues.
+			WPSEO_GSC_Settings::reload_issues();
+
+			// Adding the notification.
+			$this->add_notification( __( 'The issues have been successfully reloaded! Please refresh the page to display the new issues.', 'wordpress-seo' ), 'updated' );
```

Comment:
Are there always new issues after reloading?
If not, is it necessary to add `to display the new issues` here or could that be removed?